gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,6 +66,11 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
+      require "awesome_spawn"
+      AwesomeSpawn.run("rake evm:db:raise_failover_executed_event")
```

Comment:
I think we will need `:chdir` here. See https://github.com/ManageIQ/manageiq/blob/master/gems/pending/appliance_console/utilities.rb#L14

I'm surprised this works. Did you test this live?